src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -961,9 +943,9 @@ FixedwingAttitudeControl::task_main()
 				float speed_body_v = 0.0f;
 				float speed_body_w = 0.0f;
 				if(_att.R_valid) 	{
```

Comment:
If the rotation matrix is always valid in the control topic you can completely strip it, yes.